src/settings-ui/Settings.UI/Views/AwakePage.xaml

```diff
@@ -73,6 +73,12 @@
                         </controls:Setting.ActionContent>
                     </controls:Setting>
 
+                    <controls:Setting x:Uid="Awake_EnableAudioKeepAwake" Icon="&#xE7FB;">
```

Comment:
Can someone help me find the list of Icons I can use, as this is just a copy of the pc screen one. I'm thinking it should be a generic speaker icon. E.g. 🔉

Reply:
https://docs.microsoft.com/de-de/windows/apps/design/style/segoe-ui-symbol-font

Reply:
I think the icons are these, but I'm not sure: https://docs.microsoft.com/en-us/windows/apps/design/style/segoe-ui-symbol-font e995 is a fitting icon

Reply:
Looks so much better, thank you!
![image](https://user-images.githubusercontent.com/28542943/155741953-a99c4b7b-d9d3-4c8d-81da-c78dfdc10887.png)